src/sentry/issues/escalating.py

```diff
@@ -489,7 +489,7 @@ def manage_issue_states(
     group: Group,
     group_inbox_reason: GroupInboxReason,
     event: GroupEvent | None = None,
-    snooze_details: Mapping[str, Any] | None = None,
+    snooze_details: MutableMapping[str, Any] | None = None,
```

Comment:
`add_group_to_inbox` expects a MuttableMapping, thus, changing it.

Reply:
~ideally this should be some `TypedDict` -- but this is probably fine for now